### ISSUE TITLE
Can't change language back to English

### ISSUE THREAD

**Issue by xxyzz** [grumpygamer/DeloresDev#50]
Select any other language first then switch back to English but `loadtsvtable()` still loads the previous translation file. I'm testing on macOS 10.15.6 with engine 1.0.6.

<img width="1392" alt="Screen Shot 2020-09-25 at 10 03 38" src="https://user-images.githubusercontent.com/21101839/94218476-14943300-fed4-11ea-9edb-07ec7e62a8d5.png">

**Comment by grumpygamer** [comment]
Yes, this is a bug. It was fixed in the release I just did, but I have back-ported the fix here yet.
